fury/molecular.py

```diff
@@ -0,0 +1,698 @@
+        The molecule to which the bond is to be added.
+    atom1_index : int
+        Index of the first atom.
+    atom2_index : int
+        Index of the second atom.
+    bond_type : int (optional)
+        Type of bond (single/double/triple). Default: 1
+    """
+    molecule.AppendBond(atom1_index, atom2_index, bond_type)
+
+
+def get_total_num_atoms(molecule):
+    """Returns the total number of atoms in a given molecule.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+    """
+    return molecule.GetNumberOfAtoms()
+
+
+def get_total_num_bonds(molecule):
+    """Returns the total number of bonds in a given molecule.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+    """
+    return molecule.GetNumberOfBonds()
+
+
+def get_atomic_number(molecule, atom_index):
+    """Get the atomic number of an atom for a specified index.
+
+    Returns the atomic number of the atom present at index atom_index.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to which the atom belongs.
+    atom_index : int
+        Index of the atom whose atomic number is to be obtained.
+    """
+    return molecule.GetAtomAtomicNumber(atom_index)
+
+
+def set_atomic_number(molecule, atom_index, atomic_num):
+    """Set the atomic number of an atom for a specified index.
+
+    Assign atomic_num as the atomic number of the atom present at
+    atom_index.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to which the atom belongs.
+    atom_index : int
+        Index of the atom to whom the atomic number is to be assigned.
+    atom_num : int
+        Atomic number to be assigned to the atom.
+    """
+    molecule.SetAtomAtomicNumber(atom_index, atomic_num)
+
+
+def get_atomic_position(molecule, atom_index):
+    """Get the atomic coordinates of an atom for a specified index.
+
+    Returns the atomic coordinates of the atom present at index atom_index.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to which the atom belongs.
+    atom_index : int
+        Index of the atom whose atomic coordinates are to be obtained.
+    """
+    return molecule.GetAtomPosition(atom_index)
+
+
+def set_atomic_position(molecule, atom_index, x_coord, y_coord, z_coord):
+    """Set the atomic coordinates of an atom for a specified index.
+
+    Assign atom_coordinate to the coordinates of the atom present at
+    atom_index.
+
+    Parameters
+    ----------
+    molecule : Molecule() object
+        The molecule to which the atom belongs.
+    atom_index : int
+        Index of the atom to which the coordinates are to be assigned.
+    x_coord : float
+        x-coordinate of the atom.
+    y_coord : float
+        y-coordinate of the atom.
+    z_coord : float
+        z-coordinate of the atom.
+    """
+    molecule.SetAtomPosition(atom_index, x_coord, y_coord, z_coord)
+
+
+def get_bond_type(molecule, bond_index):
```

Comment:
Please rename "bond_type" to "bond_order", Bond type is different than bond order. A data file can have different bond types with same bond order. For example, the bond type of C-C is different than than bond type of C-H but both of them have bond order of 1.

Reply:
Thanks! Done.